src/app/static/src/app/components/plots/table/TableDisplay.vue

```diff
@@ -38,11 +39,14 @@ const defaultColDef = {
     // Stop the columns from being draggable to rearrange order or remove them
     suppressMovable: true,
 };
+const ROW_HEIGHT = 35;
```

Comment:
I reduced the row height here too, think it looks a little better

Reply:
definitely agree